addons/mail/static/src/services/new_message_service/new_message_service.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+const { Component } = owl;
+
+export const newMessageService = {
+    dependencies: ["command"],
+    start(newEnv, { command }) {
+        command.add("Message a user", () => {
+            const env = Component.env;
+            const messagingMenu = env.messaging && env.messaging.messagingMenu;
+            if (!env.messaging.device.isMobile) {
+                env.messaging.chatWindowManager.openNewMessage();
+                messagingMenu.close();
```

Comment:
I don't think it should specifically close the menu? It was the case when opening it from the menu itself by convenience, but this looks like an unrelated side effect in this case.